simulation/generateSearchQueries.js

```diff
@@ -53,29 +54,39 @@ const getRandomRoomType = () =>
 
 /* ----- HELPER FUNCTIONS END ----- */
 
-const queries = [];
 const nextSearchQueryId = incrementIdFrom(SEARCH_QUERY_ID_START);
 const nextVisitId = incrementIdFrom(VISIT_ID_START);
 const nextSearchTimestamp = countTimeFrom(new Date());
-for (let i = 0; i < DATASET_SIZE; i += 1) {
-  const { checkIn, checkOut } = getRandomDateRange();
-  const query = {
-    searchQueryId: nextSearchQueryId(),
-    timestamp: nextSearchTimestamp(),
-    visitId: nextVisitId(),
-    userId: getRandomUser(),
-    market: getRandomMarket(),
-    checkIn,
-    checkOut,
-    roomType: getRandomRoomType(),
-  };
-  queries.push(query);
-}
 
-db.emptySearchQueries()
-  .then(() => db.saveSearchQueries(queries))
-  .then(() => {
-    console.log(`SEARCH QUERY GENERATION: ${queries.length} search queries generated and saved.`);
-    db.mongoose.connection.close();
-  })
-  .catch(console.error);
+const generateQueryBatch = () => {
+  const queries = [];
+  for (let i = 0; i < BATCH_SIZE; i += 1) {
+    const { checkIn, checkOut } = getRandomDateRange();
+    const query = {
+      searchQueryId: nextSearchQueryId(),
+      timestamp: nextSearchTimestamp(),
+      visitId: nextVisitId(),
+      userId: getRandomUser(),
+      market: getRandomMarket(),
```

Comment:
This looks reasonable - will you be showing on your graphs only the data from the SF market, or all markets that were searched?

Reply:
In my Kibana dashboard, I'll probably just start with SF.